conda_build/main_develop.py

```diff
@@ -32,6 +36,12 @@ def main():
         nargs='+',
         help="Path to the source directory."
     )
+    p.add_argument(
+                   '-l', '--post-link',
```

Comment:
Oh I see the logic you've implemented now. I am thinking of the use-case where someone just types `conda develop` on a source directory and it just works. The way it is now, you have to have two steps, `conda develop` and `conda develop --post-link`. 

Would it make sense to make this enabled by default, and have `--no-post-link` and `--no-pth` flags to turn the different things off?